server/util/sendgrid_helpers.js

```diff
@@ -0,0 +1,28 @@
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendEmail = ({from_email, to_email, referral_id}) => {
+  let msg = {
+    to: `${to_email}`,
+    from: 'hatchways.tester99@gmail.com',
+    subject: 'World messenger invitation',
+    text: `${from_email} has invited you to connect on World messenger! Here’s the link to it! http://localhost:3000/join/${referral_id}`,
+  };
+  return sgMail.send(msg);
+}
+
+const sendEmailMultiple = ({from_email, to_email_ar, referral_id}) => {
```

Comment:
as per the [sendgrid npm package](https://github.com/sendgrid/sendgrid-nodejs/blob/master/docs/use-cases/multiple-emails-multiple-recipients.md) it looks like you can just do sgMail.send(messages). That also means you can probably refactor to only have one sendEmail function.